lib/smart_answer/calculators/marriage_abroad_data_query.rb

```diff
@@ -1,6 +1,6 @@
 module SmartAnswer::Calculators
```

Comment:
It happens the same in this commit: `All outcomes for Mozambique render the generic country templates` is not ideal. 
What about: `Mozambique has been flattened so there is no need to include Mozambique in any of the country lists`?